src/modules/shared/forms/components/radio-group.component.html

```diff
@@ -67,7 +67,12 @@ <h1 class="nhsuk-fieldset__heading"> {{ label }} </h1>
       <ng-container *ngIf="additional">
         <ng-container *ngFor="let addi of additional; let i = index">
           <div class="nhsuk-u-padding-top-2"></div>
-          <theme-form-input [formControl]="conditionalFormControl(addi.id)" [label]="addi.label" [description]="addi.description" [pageUniqueField]="false"></theme-form-input>
+          <ng-container *ngIf="addi.dataType != 'textarea'">

```

Comment:
I suggest changing this to "=== 'text'", to make sure that it is use properly.

If someone tries to use something else (like a radio-group), it should show nothing because it's not prepared for that.

Thoughts?

Reply:
Done as suggested